web/app.fluidity.money/app/components/NotificationSubscription.tsx

```diff
@@ -87,7 +92,9 @@ export const NotificationSubscription = ({
             ? payload.rewardType === `send`
               ? `reward for s͟e͟n͟d͟i͟n͟g`
               : `reward for r͟e͟c͟e͟i͟v͟i͟n͟g`
-            : `r͟e͟c͟e͟i͟v͟e͟d from ` + parseTrimAddress
+            : rawAddress !== payload.source
```

Comment:
it'd be good if you had an explicit check for rewardType here, but if that's too annoying that's fine

Reply:
also if you can move this block to a dedicated fn or line that'd be good